worlds/alttp/Options.py

```diff
@@ -110,13 +110,28 @@ class WorldState(Choice):
     option_inverted = 2
 
 
-class Bosses(Choice):
-    option_vanilla = 0
-    option_simple = 1
+class Bosses(TextChoice):
+    display_name = "Boss Shuffle"
+    option_none = 0
+    option_basic = 1
```

Comment:
i think you may want to have `alias_vanilla = 0` and `alias_simple = 1` ?

Reply:
I renamed them here because those options weren't actually existent in the playersettings.yaml and wasn't in the options dict so was never valid and I don't like those names anyways